jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibPluginTest.java

```diff
@@ -370,6 +373,39 @@ public void testJibTaskGroupIsSet() {
         taskName -> Assert.assertEquals(taskName, "Jib", tasks.getByPath(taskName).getGroup()));
   }
 
+  @Test
+  public void testLazyEvalForImageAndTags() {
+    try {
+      testProject.build(JibPlugin.JIB_EXTENSION_NAME);
+      Assert.fail("Expect this to fail");
+    } catch (UnexpectedBuildFailure ex) {
+      String output = ex.getBuildResult().getOutput().trim();
+
+      // Regex to parse through image and tag values from build output.
+      String cyanWordRegex = "\\u001B\\[36m(.+?)\\u001B\\[0m";
```

Comment:
I don't know if this is true, but if you use `--jib.console=plain`, it might be easier to read the output, and you can ignore the cyan coloring and stuff?

Also since we already know the image name and tags, could we just use those in the comparison directly instead of extracting them out and then using the Assert at line 404, 405?

Reply:
Thanks for the suggestion! Unfortunately, `jib.console=plain` doesn't display the progress bar but still keeps the cyan coloring:(

> Also since we already know the image name and tags, could we just use those in the comparison directly instead of extracting them out and then using the Assert at line 404, 405?
Good point, I just applied the change.

Reply:
How about passing `--console=plain` to Gradle? (Note it is not `-D...` this time.)

Reply:
Hm the cyan coloring still appears when `--console=plain` is passed to Gradle and when `Djib.console=plain` is used.

Reply:
Ugh, you're right. I see there are a couple things that are always colored.
```
Containerizing application to Docker daemon as francium25/test...
```
```
Built image to Docker daemon as francium25/test
```
I think we should fix this. Could you file a bug (and also link this issue to get rid of the interim code to handle color)?

Reply:
Yup, that's a good idea. I added a TODO with reference to issue #2764.